benchexec/tools/wit4java.py

```diff
@@ -30,25 +33,14 @@ def cmdline(self, executable, options, task, rlimits):
 
     def determine_result(self, run):
```

Comment:
I think that `determine_result` is not backwards compatible with the previous `wit4java.py`, right? Could you please add that as well? Either make the parsing generic enough that it looks for both old and new output, or set a field in `executable()` that stores the determined version and depend on that in the output parsing.

Also, `ToolNotFoundException` needs to be imported.